tests/unit/test_orgs.py

```diff
@@ -79,11 +64,7 @@ def test_create_team(self):
 
         self.post_called_with(
             url_for('teams'),
-            data={
-                'name': 'team-name',
-                'repo_names': [],
-                'permission': 'push'
-            }
+            data={'name': 'team-name', 'repo_names': [], 'permission': 'push'}
```

Comment:
Sorry @degustaf, I wasn't sufficiently clear in my earlier review. Can you revert *all* reformatting that you did like this and below. I missed these earlier and I might be missing others.